sdk/testutil/testlog.go

```diff
@@ -36,7 +37,7 @@ func NewLogBuffer(t TestingTB) io.Writer {
 	}
 	buf := &logBuffer{buf: new(bytes.Buffer)}
 	t.Cleanup(func() {
-		if t.Failed() || testing.Verbose() {
+		if t.Failed() || (!bufferVerbose && testing.Verbose()) {
```

Comment:
This can help to allow us to suppress logs from successful tests even when running `go test` with `-v`. This was useful to see how go test was parallelizing things and pass/fail as it happened without having to output all the logs.

Reply:
Nice! I've wanted to add a few more options here for hiding logs or changing the log level. This one makes sense to me.

I generally like to name the env vars something like `TEST_LOGGING_ONLY_FAILED` or something along those lines.